akka-stream/src/main/scala/akka/stream/javadsl/Graph.scala

```diff
@@ -278,10 +277,10 @@ object FlowGraph {
    * The [[Builder]] is mutable and not thread-safe,
    * thus you should construct your Graph and then share the constructed immutable [[FlowGraph]].
    */
-  def builder(): Builder = new Builder()(new scaladsl.FlowGraph.Builder)
+  def builder[Mat](): Builder[Mat] = new Builder()(new scaladsl.FlowGraph.Builder[Mat])
```

Comment:
Er, why? I really dislike one-letter type parameters except if they have a trivial meaning. We usually use In, Out, for input/output types and M or Mat for materialized types. We have many type parameters, the extra clarity helps.